app/routes/index.js

```diff
@@ -0,0 +1,61 @@
+import Ember from 'ember';
+
+const { service } = Ember.inject;
+
+export default Ember.Route.extend({
+  auth: service(),
+  tabStates: service(),
+
+  redirect() {
+    if (this.get('auth.signedIn')) {
+      if (this.get('features.dashboard')) {
+        this.transitionTo('dashboard');
+      }
+    } else if (this.get('features.enterprise')) {
+      this.transitionTo('auth');
+    }
+  },
+
+  renderTemplate(...args) {
+    if (this.get('auth.signedIn')) {
+      Ember.$('body').attr('id', 'home');
+
+      this._super(args);
+
+      this.render('repos', {
+        outlet: 'left',
+        into: 'index'
+      });
+    } else {
+      return this._super(args);
+    }
+  },
+
+  activate(...args) {
+    this._super(args);
+    if (this.get('auth.signedIn')) {
+      this.get('tabStates').set('sidebarTab', 'owned');
+      this.set('tabStates.mainTab', 'current');
+      this.controllerFor('repos').activate('owned');
+    }
+  },
+
+  deactivate() {
```

Comment:
I'm not sure if this is needed here, could you explain the reasoning behind this change?